data/store/mongo/mongo.go

```diff
@@ -913,3 +929,271 @@ func validateAndTranslateSelectors(selectors *data.Selectors) (bson.M, error) {
 
 	return selector, nil
 }
+
+// assumes all except freestyle is 5 minutes
+func getDuration(row *continuous.Continuous) int64 {
+    if strings.Contains(*row.DeviceID, "AbbottFreeStyleLibre") {
+        return 15
+    }
+    return 5
+}
+
+func (d *DataRepository) CalculateSummary(ctx context.Context, summary *summary.Summary) (*summary.Summary, error) {
+    const lowBloodGlucose = 3.9
+    const highBloodGlucose = 10
```

Comment:
magic number

Reply:
this should be/is industry standard, and currently there are no plans for it to be configurable per user, due to inconsistencies between users being unclear for clinician use.